packages/php-db-import-export/src/Backend/Assert.php

```diff
@@ -13,12 +13,16 @@ class Assert
 {
     /**
      * @param string[] $ignoreSourceColumns
+     * @param string[] $simpleLengthTypes
+     * @param string[] $complexLengthTypes
      * @throws ColumnsMismatchException
      */
     public static function assertSameColumns(
         ColumnCollection $source,
         ColumnCollection $destination,
-        array $ignoreSourceColumns = []
+        array $ignoreSourceColumns = [],
+        array $simpleLengthTypes = [],
+        array $complexLengthTypes = [],
```

Comment:
Místo assertu per backend sem přidal možnost přidat vlastní typy kde sa bude jinak porovnávat length tím je to univerzální a pro ostatní backendy stačí vypsat datové typy.